new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItWITValidation.java

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) 2020, Oracle Corporation and/or its affiliates.
+// Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+
+package oracle.weblogic.kubernetes;
+
+import oracle.weblogic.kubernetes.actions.TestActions;
+import oracle.weblogic.kubernetes.extensions.LoggedTest;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static oracle.weblogic.kubernetes.actions.TestActions.withWITParams;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+@DisplayName("Simple validation of basic WIT functions")
+class ItWITValidation implements LoggedTest {
+  @Test
+  @DisplayName("Create a MII image")
+  public void testCreatingMIIImage() {
+
+    // install WIT using the default version and location
```

Comment:
i would prefer that tests just assume it is there and use it -- let the impl worry about whether or not it needs to download it under the covers

Reply:
I'll make the change.

Reply:
The original thinking was that we'll have a place does all prerequisites. The code of downloading WIT/WDT will be moved to that place once we have it. The download code here just demonstrates how to do it.

Reply:
I can add a comment to indicate that if that's okay with you. 
Otherwise I'll move it into the impl class and remove the static access methods in TestActions. We'll have to add them back to TestActions when we have an action that does all prerequisites.